Participate/software-dashboard.md

```diff
@@ -0,0 +1,7 @@
+### Community Dashboard
+
+The community dashboard at [chaoss.biterg.io](chaoss.biterg.io) is a GrimoireLab instance provided by [Bitergia](www.bitergia.com). All contributors' information in the CHAOSS project is processed to power the dashboard. 
+
+CHAOSS community members can create, save, and share visualizations. This is encouraged in building out metrics within working groups.
+
+To request access, message [Quan Zhou](https://keybase.io/quanzh) through [Keybase](keybase.io). This allows encrypted and secure transmission of the login information.
```

Comment:
Even if we use Keybase to send the credentials, the request petition should be done following the standard procedure which is opening a Gitlab ticket at https://gitlab.com/Bitergia/c/CHAOSS/support/issues

Reply:
@sanacl I changed the request precdure as you suggested.